src/subscription_manager/managercli.py

```diff
@@ -2907,7 +2907,15 @@ def _do_command(self):
 
         syspurpose_cache = inj.require(inj.SYSTEMPURPOSE_COMPLIANCE_STATUS_CACHE)
         syspurpose_cache.load_status(self.cp, self.identity.uuid, on_date)
-        print(_("System Purpose Status: %s\n") % syspurpose_cache.get_overall_status())
+        print(_("System Purpose Status: %s") % syspurpose_cache.get_overall_status())
+
+        syspurpose_status_code = syspurpose_cache.get_overall_status_code()
+        if syspurpose_status_code != 'valid':
+            reasons = syspurpose_cache.get_status_reasons()
+            if reasons is not None:
+                for reason in reasons:
+                    print("\t%s" % reason)
```

Comment:
I feel like using a tab here is a bit too much wasted space (it uses 8 spaces):
```
+-------------------------------------------+
   System Status Details
+-------------------------------------------+
Overall Status: Invalid

Red Hat Enterprise Linux for x86_64 High Touch Beta:
- Not supported by a valid subscription.

SP Server Bits:
- Not supported by a valid subscription.

System Purpose Status: Mismatched
	unsatisfied sla: mysla
	unsatisfied role: myrole
```
I think using 2 spaces (or 4 even) would be much better.

Reply:
Yeah, you are right. I will change it and I will use `-` to match the style with list of subscriptions.